aiohttp/connector.py

```diff
@@ -964,6 +975,15 @@ async def _wrap_create_connection(
             async with ceil_timeout(
                 timeout.sock_connect, ceil_threshold=timeout.ceil_threshold
             ):
+                if self._network_interface:
+                    *args, sock_hostname, sock_port = args  # type: ignore
```

Comment:
It is important to avoid adding ignores when it's possible to make proper fixes. But if ignoring is inevitable, it shouldn't include ignoring any possible future rule violations, but only the specific rule that needs to be suppressed.

Reply:
Our mypy config disallows it anyway, it would've been sorted if we weren't fixing the code.